lib/request/address_book_query.js

```diff
@@ -10,12 +10,8 @@ var collectionQuery = require('./collection_query'),
  *   (Array.<Object>) props - list of props to request.
  */
 module.exports = function(options) {
-  return collectionQuery(
-    template.addressBookQuery({
-      props: options.props || []
```

Comment:
I think I probably wrote this for the case when we get an options without props (we really shouldn't?) but I think the template might require props?